manual/cli.rst

```diff
@@ -1,4 +1,100 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| Short     | Long                       | Description                                                                                          |
++===========+============================+======================================================================================================+
+| -r FILE   | --rivers=FILE              | produce a map of rivers, after the option it expects the name of the file where to generate the map  |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| --gs      | --grayscale-heightmap      | produce a grayscale heightmap                                                                        |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --ocean_level=N            |  elevation cut off for sea level [default = 1.0]                                                     |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --temps #/#/#/#/#/#        |  Provide alternate ranges for temperatures. [default = .126/.235/.406/.561/.634/.876]                |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --humidity #/#/#/#/#/#/#   | Provide alternate ranges for humidities. [default = .059/.222/.493/.764/.927/.986/.998]              |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| -gv N     | --gamma-value N            | N = Gamma value for temperature/precipitation gamma correction curve. [default = 1.25]               |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| -go N     | --gamma-offset N           | N = Adjustment value for temperature/precipitation gamma correction curve. [default = .2]            |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | ---not-fade-borders        | avoid fading borders                                                                                 |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --scatter                  | generate scatter plot                                                                                |
```

Comment:
An explanation or a link would be crucial in my eyes. It is *temperature vs. humidity*, I believe. So..."distribution of humidity in areas of the world plotted against their temperature" or something similar might be good.
And a link always helps, since certain areas of the plot should not be populated.